docs/dev/RELEASE.md

```diff
@@ -70,12 +70,27 @@
 9. If something goes wrong, delete the tag and release in GitHub, fix the
    problem and try again.
 
-10. Update the Homebrew formula for Dafny (see below).
+10. Make a documentation snapshot
+   a. If you have not already, make a personal clone of the 
+      dafny-lang/dafny-lang.github.io repo
+   b. Be sure that repo has a clean status
+   c. In the dafny-lang/dafny repo, execute the script
+      `docs/make-snapshot X.Y.Z`, where X.Y.Z is the new release number
+   d. Go to the Pull Request sites for both dafny-lang/dafny and
+      dafny-lang/dafny-lang.github.io and create the two PRs.
```

Comment:
Why two PRs?

Reply:
Because there are two repos.  Robin wanted to do the archiving in a separate report (the dafny-lang.github.io repo)